r1cs/gadgets/std/src/fields/nonnative/tests.rs

```diff
@@ -1402,150 +1811,178 @@ macro_rules! nonnative_test_individual {
 
 macro_rules! nonnative_test {
     ($test_name:ident, $test_simulation_field:ty, $test_constraint_field:ty) => {
+        // Commented out as the estimated number of constraints from `find_paramaters`
+        // slightly differ from the measured ones.
+        //
+        // elementary_test!(
```

Comment:
Can we remove it then ?

Reply:
I'd keep it for later. On the other hand we can always restore it from the current commit.